### ISSUE TITLE
Follow does not work

### ISSUE THREAD

**Issue by saabtech** [asimptot/FollowerBot#1]
When added username, and start listening on 127.0.0.1, nothing happens and I dont see the followers gain. Do we have to wait for a cooldown time?

**Comment by asimptot** [comment]
Hey,

You need to wait for 1000 seconds. I've just updated the code and please use the new one. If it works, let me know.

**Comment by saabtech** [comment]
> Hey,
> 
> You need to wait for 1000 seconds. I've just updated the code and please use the new one. If it works, let me know.

thanks alot, I will try it now.

**Comment by saabtech** [comment]
can you check from your end if it is working,

![image](https://user-images.githubusercontent.com/85780689/195289269-a558fb49-300e-4b5b-88d1-f3c5795c1437.png)

my followings stays the same.

**Comment by asimptot** [comment]
Did you get followers on your account? If no, let me check.

**Comment by asimptot** [comment]
Please share your username and then let me check on my side for getting followers.

**Comment by saabtech** [comment]
> Please share your username and then let me check on my side for getting followers.

username: talabat_mandob

**Comment by asimptot** [comment]
did you get 10 followers, now? If so, let me know and send your logs to me.

**Comment by saabtech** [comment]
Not yet here is the screenshot:

![image](https://user-images.githubusercontent.com/85780689/195338599-875877fb-e471-4017-948e-8741f2d765f7.png)


> did you get 10 followers, now? If so, let me know and send your logs to me.

**Comment by asimptot** [comment]
Hey,

I checked you have just gained 10 followers. Please check your account and please try to run up to date code. I have just added success message. The system can be slow now. Don't terminate running the program.

**Comment by saabtech** [comment]
> Hey,
> 
> I checked you have just gained 10 followers. Please check your account and please try to run up to date code. I have just added success message. The system can be slow now. Don't terminate running the program.

Yes, I received the followers now below screenshot... Noted i will check it and run it.

![image](https://user-images.githubusercontent.com/85780689/195345051-26ac8009-5746-4b07-beb2-cc329088f448.png)

**Comment by saabtech** [comment]
not sure what the problem is, I get this error :

![image](https://user-images.githubusercontent.com/85780689/195353968-d8591f2f-cab3-4125-a457-4884cda27332.png)


the follower doesnt work with me.. I am using windows 10 pro. do i need any additional drivers or something cuz it might be due to that

**Comment by asimptot** [comment]
We ran the program for the same account simultaneosly, so this is why you got the error. Please try it for tomorrow. As far as i saw, you gained 100 followers approximately!

**Comment by saabtech** [comment]
> We ran the program for the same account simultaneosly, so this is why you got the error. Please try it for tomorrow. As far as i saw, you gained 100 followers approximately!

It does not work with me, i get error occured, I did get 100 followers from your end, but I tried all possible ways. Can you tell us the instruction related to the additional information such as OS, drivers for selenium / chrome , script working on windows or Linux? so I can narrow my search and work around to find the solutions

**Comment by EdenEitani** [comment]
> > We ran the program for the same account simultaneosly, so this is why you got the error. Please try it for tomorrow. As far as i saw, you gained 100 followers approximately!
> 
> It does not work with me, i get error occured, I did get 100 followers from your end, but I tried all possible ways. Can you tell us the instruction related to the additional information such as OS, drivers for selenium / chrome , script working on windows or Linux? so I can narrow my search and work around to find the solutions

You should remove the `chrome_options.add_argument('--headless')` option from init.py and see what is the error. You might get the same as I commented here ^^

**Comment by asimptot** [comment]
@saabtech hi,

Those are my pip list. Please install the correct version on your setup.

Package                Version
---------------------- ------------
aicrowd-api            0.1.23
antlr4-python3-runtime 4.9.3
async-generator        1.10
async-timeout          4.0.2
attrs                  21.4.0
audio2numpy            0.1.2
beautifulsoup4         4.11.1
behave                 1.2.6
bs4                    0.0.1
certifi                2022.6.15
cffi                   1.15.1
charset-normalizer     2.1.0
click                  8.1.3
cloudpickle            2.1.0
colorama               0.4.5
colorlog               6.6.0
contourpy              1.0.5
cryptography           37.0.4
cycler                 0.11.0
Cython                 0.29.32
decorator              4.4.2
demucs                 3.0.5
Deprecated             1.2.13
diffq                  0.2.2
dora-search            0.1.10
ffmpeg                 1.4
ffmpeg-python          0.2.0
flake8                 5.0.4
fonttools              4.37.2
future                 0.18.2
h11                    0.13.0
hydra-colorlog         1.2.0
hydra-core             1.2.0
idna                   3.3
imageio                2.21.1
imageio-ffmpeg         0.4.7
iniconfig              1.1.1
irisvmpy               0.0.1
joblib                 1.2.0
jsonschema             4.12.1
julius                 0.2.6
kiwisolver             1.4.4
lameenc                1.4.1
matplotlib             3.6.0
mccabe                 0.7.0
mouse                  0.7.1
MouseInfo              0.1.3
moviepy                1.0.3
musdb                  0.4.0
museval                0.4.0
mypy                   0.971
mypy-extensions        0.4.3
numpy                  1.23.2
omegaconf              2.2.2
openunmix              1.2.1
outcome                1.2.0
packaging              21.3
pandas                 1.4.3
parse                  1.19.0
parse-type             0.6.0
Pillow                 9.2.0
pip                    22.2.2
pluggy                 1.0.0
proglog                0.1.10
py                     1.11.0
pyaml                  21.10.1
PyAutoGUI              0.9.53
pycodestyle            2.9.1
pycparser              2.21
pydub                  0.25.1
pyflakes               2.5.0
PyGetWindow            0.0.9
PyMsgBox               1.0.9
pyOpenSSL              22.0.0
pyparsing              3.0.9
pyperclip              1.8.2
PyRect                 0.2.0
pyrsistent             0.18.1
PyScreeze              0.1.28
PySocks                1.7.1
PySoundFile            0.9.0.post1
pystoi                 0.3.3
pytest                 7.1.3
python-dateutil        2.8.2
python-dotenv          0.20.0
pytweening             1.0.4
pytz                   2022.2.1
PyYAML                 6.0
redis                  4.3.4
requests               2.28.1
retrying               1.3.3
scikit-learn           1.1.2
scipy                  1.9.0
selenium               4.3.0
setuptools             58.1.0
simplejson             3.17.6
six                    1.16.0
sklearn                0.0
sniffio                1.2.0
sortedcontainers       2.4.0
SoundFile              0.10.3.post1
soupsieve              2.3.2.post1
stempeg                0.2.3
submitit               1.4.5
threadpoolctl          3.1.0
tomli                  2.0.1
torch                  1.12.1
torchaudio             0.12.1
torchmetrics           0.9.3
tqdm                   4.64.0
treetable              0.2.5
trio                   0.21.0
trio-websocket         0.9.2
typing_extensions      4.3.0
urllib3                1.26.11
webdriver-manager      3.8.2
wrapt                  1.14.1
wsproto                1.1.0

@EdenEitani hi,

I had this issue and trying to figure out this. Please open a different issue to make fix it. It means all followers in the system followed you and there was no enough followers.

**Comment by saabtech** [comment]
> chrome_options.add_argument('--headless')

ok,... when i run the script, the page opens, scroll down and click on some thing ( sorry i am not turkish ), move to this page "https://tolinay.com/tiktok-begeni-hilesi) " and stay there and does nothing.

**Comment by EdenEitani** [comment]
> > chrome_options.add_argument('--headless')
> 
> ok,... when i run the script, the page opens, scroll down and click on some thing ( sorry i am not turkish ), move to this page "https://tolinay.com/tiktok-begeni-hilesi) " and stay there and does nothing.

It happened to me too. I've changed some things. 
Will open a pull request for @asimptot to approve.

**Comment by asimptot** [comment]
@saabtech hi,

You don't have to know Turkish. Please update your libraries according to version that I shared. 

@EdenEitani hi,

You got different error. @saabtech cannot enter even his username to textbox.

**Comment by EdenEitani** [comment]
@asimptot, Of course, I know that, but I encountered his problem too.
I've opened a PR that probably solves this issue

**Comment by asimptot** [comment]
@EdenEitani , @saabtech hi,

I have just updated the code and added xpath line instead of tab controls. Please let me know if it's fixed on your side.

**Comment by asimptot** [comment]
Please use up-to-date code for this issue.
